.github/workflows/build.yml

```
@@ -0,0 +1,123 @@
+name: Build and upload to PyPI
+# publish when a (published) GitHub Release is created
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels:
+    name: Build wheels [${{ matrix.os }}, ${{ matrix.platform }}, py ${{ matrix.python-version }}]
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        platform: [x64]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Set up python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.6.3
+      - name: Install Visual C++ for Python 2.7
+        if: runner.os == 'Windows'
+        run: |
+          choco install vcpython27 -f -y
+
+      - name: Build wheels on ${{ matrix.os}}
+        run: |
+          RUNNER_OS="${{ runner.os }}"
+          PLATFORM="${{ matrix.platform }}"
+          PYTHON_VERSION="${{ matrix.python-version }}"
```

If you get rid of the '.', you could use this directly in the following code without needing all the `if`

```suggestion
          PYTHON_VERSION=$(echo ${{ matrix.python-version }} | tr -d '.')
```

But see suggestion below to avoid the need for PYTHON_VERSION

---

agreed. my bash skills are lacking...